setup.py

```diff
@@ -56,6 +56,7 @@
         'pyramid == 1.7',
         'pysensu-yelp >= 0.2.2',
         'pytimeparse >= 1.1.0',
+        'pytz == 2014.10',
```

Comment:
Do we really need this specific version? I imagine any recent version would be sufficient. Can you switch this to a `>=` relationship? Timezone databases change all the time, so supporting more recent versions of the module would be preferred.